example/node.ts

```diff
@@ -0,0 +1,3 @@
+import { System } from '../src';
+
+System.import('./node-target.ts');
```

Comment:
This file is similar to `bin/requirex` so if launch uses that, the minimal example has one less file assuming package users will also have the same binary installed.

Reply:
My thinking was that it makes it clear exactly what's going on. Rather than some "magic" bin being used...
Also, this allows the full integrated debugging using ts-node